broker/server.js

```diff
@@ -104,102 +104,127 @@ app.post("/Encounter", (req, res) => {
     .catch((e) => res.send(e));
 });
 
+// Discharge
+app.get("/discharge", (req, res) => {
```

Comment:
In a RESTful API, GET requests should not change the state of a resource. Could you make this a POST request instead (same thing with check-in and any others I'm missing)? I don't know if we are aiming for RESTful in this project, but that would make more sense for those of us used to RESTful APIs.

Reply:
@rrucker3 I'd say it'd be a good idea to follow RESTful conventions, yes. I don't think we're trying to follow it 100% to the tee however